experiments/ssl4eo/landsat/conf/ssl4eo_benchmark_etm_sr_cdl.yaml

```diff
@@ -22,4 +22,24 @@ data:
     root: "data/ssl4eo_benchmark"
     sensor: "etm_sr"
     product: "cdl"
-    classes: [0, 1, 5, 24, 36, 37, 61, 111, 121, 122, 131, 141, 142, 143, 152, 176, 190, 195]
+    classes:
```

Comment:
Unfortunately this is the way how Prettier is handling lists. This topic was discussed with authors already 5-6 years ago and there are still no changes, but there are 3 solutions:
- adding `// prettier-ignore` on the lines which would be quite troublesome 
- adding [prettier-plugin-multiline-arrays](https://github.com/electrovir/prettier-plugin-multiline-arrays) plugin to Prettier, which is a better solution, but it requires installing this plugin
- last one is to leave it as it is if there is not too many cases with such a long arrays that could be hard to read

Reply:
If you format it this way manually, will prettier reformat it back to square brackets?

Reply:
Yes, every time it will go through CI it will be set to one-liner. Only solution to keep them in multi-line format as I mention is to use `prettier-ignore` above those lines or install a plugin.

Reply:
Are you sure? `tests/conf/seco_byol_1.yaml` is formatted as a list without square brackets.

Reply:
I think we had a misunderstanding but now I exactly get what you mean.

Yes if the file will be initially formatted like:
```yaml
classes:
   - 1
   - 2
   - 3
```
It will stay that way. I assume that `experiments/ssl4eo/landsat/conf/ssl4eo_benchmark_etm_sr_cdl.yaml` was formatted this way due to maximum line length.

I see that it's one of the flaws of Prettier that it will not unify such cases because if it will be initially formatted in this way:
```yaml
classes:
   - 1
   - 2
   - 3
```
it will stay that way, and if it will be in such format:
```yaml
[0, 11, 21, 22, 31, 41, 42, 43, 52, 71, 81, 82, 90, 95]
```
it will also gonna be keep that way unless it will exceed max line length which can be adjusted in `.toml` file then it will look like `experiments/ssl4eo/landsat/conf/ssl4eo_benchmark_etm_sr_cdl.yaml`

Reply:
In that case, can we format these files like:
```yaml
classes:
   - 1
   - 2
   - 3
```

Reply:
Yes but it all will have to be done manually, so if it will get some new files and changes formatted this way:
```yaml
[0, 11, 21, 22, 31, 41, 42, 43, 52, 71, 81, 82, 90, 95]
```
it will stay that way.

I already started manually changing files.